src/styles/header.scss

```diff
@@ -0,0 +1,119 @@
+.header {
+  @include desktop {
+    max-width: 1280px;
```

Comment:
The header element should not have a max-width. Its width will be determined by its containing element.